app/Http/Controllers/PhotosController.php

```diff
@@ -333,7 +333,7 @@ public function addTags (AddTagsRequest $request, AddCustomTagsToPhotoAction $cu
 
         $this->updateLeaderboardsAction->run($photo, $user->id, $litterTotals['all'] + $customTagsTotal);
 
-        $photo->remaining = !$request->presence;
+        $photo->remaining = !$request->picked_up;
```

Comment:
need to double check what value is being passed here. Would be good to update the column name to picked up...